community/terraform/APKBUILD

```diff
@@ -20,12 +21,18 @@ prepare() {
 
 build() {
 	cd "$builddir"
-	export GOPATH="$srcdir"
-	go build -v -o bin/$pkgname
+	GOPATH="$srcdir" go build -v -o bin/$pkgname \
+		-ldflags "-X main.GitCommit=v$pkgver -X github.com/hashicorp/terraform/version.Prerelease= -s -w"
```

Comment:
For reviewers; this follows upstream procedures [done in their release builds](https://github.com/hashicorp/terraform/blob/master/scripts/build.sh#L46..L48).